### PR TITLE
[api/kitsu] Handle manga metadatas properly

### PR DESCRIPTION
**Pull request by ahmubashshir**
* Use proper filter for manga.
* Handle anime/manga specific metadata.

### CODE REVIEW

File: trackma/lib/libkitsu.py [hunk 1]
```diff
@@ -304,11 +310,21 @@ def fetch_list(self):
...
                 ]),
                 "page[limit]": "250",
             }
 
+            if self.mediatype == 'anime':
+                params['fields[anime]'] = params['fields[anime]'] + ',nsfw'
+
+            if self.mediatype == 'manga':
+                params['fields[manga]'] = params['fields[manga]'] + \
+                    ',serialization'
```
Comment (FichteFoll) [inline trackma/lib/libkitsu.py:326]:
```suggestion
                params['fields[anime]'] += ',nsfw'

            if self.mediatype == 'manga':
                params['fields[manga]'] += ',serialization'
```

File: trackma/lib/libkitsu.py [hunk 2]
```diff
@@ -549,29 +565,36 @@ def _parse_info(self, media):
...
+                ('Rank',                "Popularity #{}\nRating #{}".format(
+                    attr.get('popularityRank', '?'), attr.get('ratingRank', '?'))),
+                ('Age Rating',          "{} ({})".format(
+                    attr.get('ageRating', 'Unknown'), attr.get('ageRatingGuide', 'Unknown'))),
             ]
         })
 
+        if attr.get('nsfw', False):
+            info['extra'].insert(
+                2, ('NSFW', 'Yes'))
```
Comment (FichteFoll) [inline trackma/lib/libkitsu.py:588]: Why do you wrap lines so aggressively?

Comment (ahmubashshir) [reply trackma/lib/libkitsu.py:588]: [autopep8](https://github.com/ahmubashshir/dotfiles/blob/master/dot-config:git/hooks/hooks.d/pre-commit/format)'s fault :3

Comment (FichteFoll) [reply trackma/lib/libkitsu.py:587]: I suggest to disable such a tool for pull requests to other projects, as this introduces uncecessary and often undesired changes, not unlike the one I highlighted above. It has already caused me to manually resolve conflicts in an open PR of mine in multiple files and was quite a bother.

This isn't my project, so I'm not going to campaign for or against certain code style choices, but I consider a maximum line length below 100 to be counterproductive, and doubly so if the line breaks are done arbitrarily and without a cohesive pattern (and on a line with a width of 52). If you want automatic formatting, `black` is most likely the better choice.

Comment (ahmubashshir) [reply trackma/lib/libkitsu.py:587]: ok, I'll set it to run if it's explicitly enabled.

**Pull request merged**